.github/workflows/supported_storages_compatibility_check.yaml

```diff
@@ -0,0 +1,728 @@
+
+      - name: Dockerfile Lint for ScalarDB Server
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: ':server:dockerfileLint'
+
+      - name: Dockerfile Lint for ScalarDB Schema Loader
+        if: always()
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: ':schema-loader:dockerfileLint'
+
+  cassandra-2-2-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:2.2.19
+        env:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_2.2_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cassandra-3-0-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:3.0.29
+        env:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_3.0_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cassandra-3-11-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:3.11.15
+        env:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_3.11_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cosmos-integration-test:
```

Comment:
Maybe we don't need this integration test for Cosmos since we already check it in the usual integration tests.

Reply:
Right, I will delete all the storage versions that are already listed in the CI.